### ISSUE TITLE
Documentation states that HTTP Basic Auth is used, but it isn't

### ISSUE THREAD

**Issue by ryanmoran** [cloudfoundry/uaa#179]
The documentation states that the GET /token_key endpoint is protected by HTTP Basic Auth, but this is not actually the case:

```bash
→ curl -v -k https://uaa.example.cf-app.com/token_key
* Hostname was NOT found in DNS cache
*   Trying 52.7.245.225...
* Connected to uaa.example.cf-app.com (52.7.245.225) port 443 (#0)
* TLS 1.2 connection using TLS_RSA_WITH_AES_128_CBC_SHA256
* Server certificate: *.10.244.0.34.xip.io
> GET /token_key HTTP/1.1
> User-Agent: curl/7.37.1
> Host: uaa.example.cf-app.com
> Accept: */*
>
< HTTP/1.1 200 OK
< Access-Control-Allow-Origin: *
< Cache-Control: no-cache, no-store, max-age=0, must-revalidate
< Content-Type: application/json;charset=UTF-8
< Date: Mon, 29 Jun 2015 18:22:54 GMT
< Expires: 0
< Pragma: no-cache
* Server Apache-Coyote/1.1 is not blacklisted
< Server: Apache-Coyote/1.1
< Strict-Transport-Security: max-age=31536000 ; includeSubDomains
< X-Cf-Requestid: c33c734c-e588-4122-656e-938454fcdfad
< X-Content-Type-Options: nosniff
< X-Frame-Options: DENY
< X-Xss-Protection: 1; mode=block
< Content-Length: 526
<
* Connection #0 to host uaa.mrorange.cfla.cf-app.com left intact
{"alg":"SHA256withRSA","value":"-----BEGIN PUBLIC KEY-----\nMIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDHFr+KICms+tuT1OXJwhCUmR2d\nKVy7psa8xzElSyzqx7oJyfJ1JZyOzToj9T5SfTIq396agbHJWVfYphNahvZ/7uMX\nqHxf+ZH9BL1gk9Y6kCnbM5R60gfwjyW1/dQPjOzn9N394zd2FJoFHwdq9Qs0wBug\nspULZVNRxq7veq/fzwIDAQAB\n-----END PUBLIC KEY-----\n","kty":"RSA","use":"sig","n":"AMcWv4ogKaz625PU5cnCEJSZHZ0pXLumxrzHMSVLLOrHugnJ8nUlnI7NOiP1PlJ9Mirf3pqBsclZV9imE1qG9n/u4xeofF/5kf0EvWCT1jqQKdszlHrSB/CPJbX91A+M7Of03f3jN3YUmgUfB2r1CzTAG6CylQtlU1HGru96r9/P","e":"AQAB"}
```

**Comment by fhanik** [comment]
It is protected if your signing key is the same as your verification key - I will update the docs accordingly

**Comment by fhanik** [comment]
Fixed via 8d7632bc9ba2a6abd253d3cac8feeaa17838eab7
